app/src/main/java/org/kiwix/kiwixmobile/utils/KiwixTextToSpeech.java

```diff
@@ -135,14 +152,31 @@ public void stop() {
     }
   }
 
+  public Boolean requestAudioFocus() {
+    int audioFocusRequest = am.requestAudioFocus(onAudioFocusChangeListener, AudioManager.STREAM_MUSIC, AudioManager.AUDIOFOCUS_GAIN);
+
+    Log.d(TAG_KIWIX, "Audio Focus Requested");
+
+    synchronized (focuslock) {
+      if (audioFocusRequest == AudioManager.AUDIOFOCUS_REQUEST_GRANTED) {
+        return true;
+      } else {
+        return false;
+      }
+    }
+  }
+
   public void pauseOrResume() {
     if (currentTTSTask == null)
       return;
 
-    if (currentTTSTask.paused)
+    if (currentTTSTask.paused) {
```

Comment:
Could we convert this to an if/else if. This is really messy. Remember spacing: } else {